src/index.js

```diff
@@ -78,9 +78,8 @@ module.exports = exports = options => {
    */
 
   async function empty(room) {
-    if (!rooms.get(room)) return
-    if (!Array.isArray(room)) clear(room)
-    room.forEach(clear)
+    if (Array.isArray(room)) room.forEach(clear)
+    if (rooms.get(room)) clear(room)
```

Comment:
I would use `rooms.has(room)` and use `else if` so it is not checked if `room` is an array.